test/test_decoding.py

```diff
@@ -0,0 +1,144 @@
+import tensorflow as tf
+
+from wavenet import WaveNet
+from test_encoding import manual_encode
+import json
+import numpy as np
+
+def manual_decode(signal, channels):
+    # Calculate inverse mu-law companding and dequantization
+    mu = channels
+    output = signal.astype(np.float32)
+    y = (2*output - 1) / mu
+    x = np.sign(y) * (np.exp(y * np.log(1 + mu)) - 1) / mu
+
+    return x
+
+
+class TestDecode(tf.test.TestCase):
+
+    def setUp(self):
+        quantization_steps=256
+        self.net = WaveNet(batch_size = 1,
```

Comment:
The spacing around the `=` is inconsistent here. I would personally prefer no spaces around `=` for kwargs, like in PEP8.